hack/local-up-cluster.sh

```diff
@@ -947,6 +947,15 @@ EOF
 fi
 }
 
+# If we are running in the CI, we need a few more things before we can start
+if [[ "${KUBETEST_IN_DOCKER:-}" == "true" ]]; then
+  echo "Preparing to test ..."
+  ${KUBE_ROOT}/hack/install-etcd.sh
+  export PATH="${KUBE_ROOT}/third_party/etcd:${PATH}"
+  KUBE_FASTBUILD=true make ginkgo cross
```

Comment:
why cross compiling?

Reply:
the CI system is basically running kubetest with the "local" deployment option (https://github.com/kubernetes/test-infra/blob/master/kubetest/local.go) and it needs the `make cross` for the end-to-end conformance test to work properly.